### PR TITLE
feat: Support pnpm_version input

### PR DESCRIPTION
**Pull request by bflad**
This input enables pnpm to be globally installed into the Docker runtime. It is not recommended for use without consulting Speakeasy support.

### CODE REVIEW

**Review by disintegrator — APPROVED** [review]

File: action.yml [hunk 1]
```diff
@@ -91,6 +91,9 @@ inputs:
     description: |
       Extra environment variables to set for the speakeasy run execution.
     required: false
+  pnpm_version:
```
Comment (ryan-timothy-albert) [inline action.yml:94]: Should need to also add this to workflow-executor.yaml otherwise they won't be able to pass it through

Comment (bflad) [reply action.yml:94]: Should be covered by https://github.com/speakeasy-api/sdk-generation-action/pull/168/commits/79aeb68c7e7cd770737cfdb3f93d98e2493908d2 -- please let me know if there's anything else there

Comment (ryan-timothy-albert) [reply action.yml:94]: needs to also add inputs.pnpm_version at the bottom of this file

File: .github/workflows/workflow-executor.yaml [hunk 2]
```diff
@@ -56,7 +56,7 @@ on:
       environment:
         description: |
           Any required environment variables to set for the speakeasy run execution. Set as a string with each entry of the form "key=value".
-  
+
```
Comment (ryan-timothy-albert) [inline .github/workflows/workflow-executor.yaml:59]: intentional?

**Review by ryan-timothy-albert — APPROVED** [review]
I think it's fine just to release this as new V15. I would just test it ourselves to ensure it installs pnpm correctly before we tell Vercel

File: .github/workflows/workflow-executor.yaml (hunk 2 above)
Comment (bflad) [reply .github/workflows/workflow-executor.yaml:59]: Guessing that was my editor automatic formatting. Doesn't seem like the whitespace there would have any effect

**Pull request merged**